### ISSUE TITLE
Standardize dates across the app 

### ISSUE THREAD

**Issue by isabelcshaw** [OpenSRP/opensrp-client-chw#1238]
Ideally, across the application all dates should show as: 
**DD/3-letter-month/YYYY**

There are three places this is inconsistent: 
1. After using the date selector dialog in forms, the field entry becomes DD-MM-YYYY format. 
2. The date selector in the "Record vaccinations" view 
3. Dates in medical history and due/overdue on the child profile 

Based on what Roger shared with me, it sounds like the last internal conversation on this determined that the hardest part is actually this first problem. @rkodev do you have any context here? if so, does that sound correct?

**Comment by rkodev** [comment]
@isabelcshaw yes, the first part of the problem would need some modification to the Date factory in Native forms. It might take 3-4 days for the first part of the issue to be completed.
The other parts of the app should take 2-3 days to be completed

**Comment by isabelcshaw** [comment]
@rkodev ok, super helpful, thank you. Let's implement 2 and 3 then and hold on 1 for now until we see how much progress we're making at the end of next week
